### ISSUE TITLE
It's not working

### ISSUE THREAD

**Issue by BlizzardWolfang** [GamerGirlandCo/Songsterr-Plus-2.0#29]

**Comment by ksyushabel** [comment]
> Have you solved it?

**Comment by jcf-junior** [comment]
Yes. Install Tampermonkey (it's a browser extension that runs scripts)

Then install this script:
https://update.greasyfork.org/scripts/469079/Songsterr%20Premium%20-%20songsterrcom.user.js

I am not the creator of the script, I am just reposting this.
